lib/hooks/useTopicOptions.ts

```diff
@@ -0,0 +1,7 @@
+const useTopicOptions = () => {
```

Comment:
By convention, this should not be prefixed with `use` as it is not a hook, but a function that returns data. It should also be moved to `lib/utils`

```suggestion
const getTopicOptions = () => {
```